sentry/src/test/java/io/sentry/SentryClientTest.kt

```diff
@@ -1592,6 +1593,234 @@ class SentryClientTest {
         )
     }
 
+    @Test
+    fun `can pass an attachment via hints`() {
+        val sut = fixture.getSut()
+
+        sut.captureException(IllegalStateException(), Hints.withAttachment(fixture.attachment))
+
+        thenEnvelopeIsSentWith(1, 0, 1)
```

Comment:
`l`: I had to scroll down all the way to understand what these numbers stand for. Maybe it makes to specify the argument names explicitly to improve readability.

Reply:
If you're in an IDE, it'll show pretty much the same way as a named parameter call but I can use the param names.

Reply:
Updated the code